apps/backend/src/libs/modules/config/base-config.module.ts

```diff
@@ -87,6 +87,20 @@ class BaseConfig implements Config {
 					format: Number,
 				},
 			},
+			MAPBOX: {
+				MAPBOX_ACCESS_TOKEN: {
+					default: null,
+					doc: "Mapbox api access token",
+					env: "MAPBOX_ACCESS_TOKEN",
+					format: String,
+				},
+				MAPBOX_URL: {
+					default: null,
+					doc: "Mapbox URL",
+					env: "MAPBOX_URL",
+					format: String,
+				},
```

Comment:
let's unitfy the naming in groups to avoid repetition
```suggestion
			MAPBOX: {
				ACCESS_TOKEN: {
					default: null,
					doc: "Mapbox api access token",
					env: "MAPBOX_ACCESS_TOKEN",
					format: String,
				},
				BASE_URL: {
					default: null,
					doc: "Mapbox base URL",
					env: "MAPBOX_BASE_URL",
					format: String,
				},
```